documentation/workflows.md

```diff
@@ -0,0 +1,74 @@
+# Working on the Project
+
+Better the be explicit/specific on topics than to wait for avoidable confusion due to ambiguity.
+
+## Principles
+
+1. No work is ever done directly on `main`; ALWAYS branch off of the target receiving branch (i.e. `main` for the majority of work)
+2. All changes must be reviewed and approved, and pass all quality gates
+
+## Development Workflows
+
+- [Starting Work](#starting-work)
+- [Pipelines](#pipelines)
+- [Changesets](#changesets)
+- [Approvals](#approvals)
+- [Local Testing](#local-testing)
+
+### Starting Work
+
+1. Pick a task from the issues/project
+    - Ask the team if anyone is working on the task
```

Comment:
how should people ask? what are the communication channels?

Reply:
More detail added now.